spec/middleware/sync-storage-manager-spec.js

```diff
@@ -1,6 +1,8 @@
 const beforeEachHelpers = require('../helpers/before-each');
 
 describe('sync-brainstem middleware', () => {
+  const extractArgsFromCalls = method => method.calls.all().map(call => call.args[0]);
```

Comment:
There is native support for this on Jasmine

Reply:
Ok, will try that out.